sr_utilities/scripts/sr_utilities/controller_spawner.py

```diff
@@ -17,44 +17,50 @@
 from __future__ import absolute_import
 import os
 import re
+import sys
 import rospkg
 import rospy
 import yaml
-from controller_manager_msgs.srv import ListControllers
-from controller_manager_msgs.srv import SwitchController, LoadController
+from controller_manager_msgs.srv import (ListControllers, LoadController,
+                                         SwitchController, SwitchControllerRequest)
 from sr_utilities.hand_finder import HandFinder
 
 
-class ControllerSpawner(object):
+class ControllerSpawner:
     JOINT_NAMES = ["rh_FFJ1", "rh_FFJ2", "rh_FFJ3", "rh_FFJ4", "rh_LFJ1", "rh_LFJ2", "rh_LFJ3", "rh_LFJ4", "rh_LFJ5",
                    "rh_MFJ1", "rh_MFJ2", "rh_MFJ3", "rh_MFJ4", "rh_RFJ1", "rh_RFJ2", "rh_RFJ3", "rh_RFJ4", "rh_THJ1",
                    "rh_THJ2", "rh_THJ3", "rh_THJ4", "rh_THJ5", "rh_WRJ1", "rh_WRJ2",
                    "lh_FFJ1", "lh_FFJ2", "lh_FFJ3", "lh_FFJ4", "lh_LFJ1", "lh_LFJ2", "lh_LFJ3", "lh_LFJ4", "lh_LFJ5",
                    "lh_MFJ1", "lh_MFJ2", "lh_MFJ3", "lh_MFJ4", "lh_RFJ1", "lh_RFJ2", "lh_RFJ3", "lh_RFJ4", "lh_THJ1",
                    "lh_THJ2", "lh_THJ3", "lh_THJ4", "lh_THJ5", "lh_WRJ1", "lh_WRJ2"]
 
-    def __init__(self, config_file_path, service_timeout, excluded_joints=[]):
-        self._config_file_path = config_file_path
-        self._service_timeout = service_timeout
+    def __init__(self, config_file_path_, service_timeout_, excluded_joints_=None):
```

Comment:
I think the trailing `_` suffix is only recommended for avoiding conflicts with python keywords, not variable names that we have full control of!

Reply:
good point about trailing `_` being for python keywords. I've added leading underscores in the main instead, thoughts? 7113a011792b2cdba54e7bb8845356298efb715b

Reply:
I don't usually prepend `__main__`-scope variables with `_`, just class member variables that I want to indicate are "protected". Using it in `__main__` doesn't do any harm, but might confuse people?

Reply:
https://google.github.io/styleguide/pyguide.html#3162-naming-conventions

Reply:
922a19db3bc6e43f71f1e48d736f2eae0b2d9ba5

Reply:
Looks good to me